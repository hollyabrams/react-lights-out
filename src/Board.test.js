import { render } from '@testing-library/react';
import Board from './Board';

test('renders Board without crashing', () => {
    render(<Board nrows={5} ncols={5} chanceLightStartsOn={0.5} />);
});
  
 
test('renders Board correctly', () => {
    const { container } = render(<Board nrows={5} ncols={5} chanceLightStartsOn={0.5} />);
    expect(container.firstChild).toMatchSnapshot();
});