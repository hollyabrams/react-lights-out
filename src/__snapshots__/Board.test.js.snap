// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Board correctly 1`] = `
<table
  class="Board"
>
  <tbody>
    <tr>
      <td>
        <td
          class="Cell Cell-lit"
        >
          <span
            class="lightbulb"
          >
            ðŸ’¡
          </span>
        </td>
      </td>
      <td>
        <td
          class="Cell "
        >
          <span
            class="lightbulb"
          />
        </td>
      </td>
      <td>
        <td
          class="Cell "
        >
          <span
            class="lightbulb"
          />
        </td>
      </td>
      <td>
        <td
          class="Cell "
        >
          <span
            class="lightbulb"
          />
        </td>
      </td>
      <td>
        <td
          class="Cell Cell-lit"
        >
          <span
            class="lightbulb"
          >
            ðŸ’¡
          </span>
        </td>
      </td>
    </tr>
    <tr>
      <td>
        <td
          class="Cell "
        >
          <span
            class="lightbulb"
          />
        </td>
      </td>
      <td>
        <td
          class="Cell Cell-lit"
        >
          <span
            class="lightbulb"
          >
            ðŸ’¡
          </span>
        </td>
      </td>
      <td>
        <td
          class="Cell Cell-lit"
        >
          <span
            class="lightbulb"
          >
            ðŸ’¡
          </span>
        </td>
      </td>
      <td>
        <td
          class="Cell Cell-lit"
        >
          <span
            class="lightbulb"
          >
            ðŸ’¡
          </span>
        </td>
      </td>
      <td>
        <td
          class="Cell "
        >
          <span
            class="lightbulb"
          />
        </td>
      </td>
    </tr>
    <tr>
      <td>
        <td
          class="Cell Cell-lit"
        >
          <span
            class="lightbulb"
          >
            ðŸ’¡
          </span>
        </td>
      </td>
      <td>
        <td
          class="Cell Cell-lit"
        >
          <span
            class="lightbulb"
          >
            ðŸ’¡
          </span>
        </td>
      </td>
      <td>
        <td
          class="Cell Cell-lit"
        >
          <span
            class="lightbulb"
          >
            ðŸ’¡
          </span>
        </td>
      </td>
      <td>
        <td
          class="Cell "
        >
          <span
            class="lightbulb"
          />
        </td>
      </td>
      <td>
        <td
          class="Cell Cell-lit"
        >
          <span
            class="lightbulb"
          >
            ðŸ’¡
          </span>
        </td>
      </td>
    </tr>
    <tr>
      <td>
        <td
          class="Cell Cell-lit"
        >
          <span
            class="lightbulb"
          >
            ðŸ’¡
          </span>
        </td>
      </td>
      <td>
        <td
          class="Cell Cell-lit"
        >
          <span
            class="lightbulb"
          >
            ðŸ’¡
          </span>
        </td>
      </td>
      <td>
        <td
          class="Cell Cell-lit"
        >
          <span
            class="lightbulb"
          >
            ðŸ’¡
          </span>
        </td>
      </td>
      <td>
        <td
          class="Cell "
        >
          <span
            class="lightbulb"
          />
        </td>
      </td>
      <td>
        <td
          class="Cell Cell-lit"
        >
          <span
            class="lightbulb"
          >
            ðŸ’¡
          </span>
        </td>
      </td>
    </tr>
    <tr>
      <td>
        <td
          class="Cell "
        >
          <span
            class="lightbulb"
          />
        </td>
      </td>
      <td>
        <td
          class="Cell "
        >
          <span
            class="lightbulb"
          />
        </td>
      </td>
      <td>
        <td
          class="Cell "
        >
          <span
            class="lightbulb"
          />
        </td>
      </td>
      <td>
        <td
          class="Cell "
        >
          <span
            class="lightbulb"
          />
        </td>
      </td>
      <td>
        <td
          class="Cell "
        >
          <span
            class="lightbulb"
          />
        </td>
      </td>
    </tr>
  </tbody>
</table>
`;
